software/HeartRateDetection/HeartRateDetection.ino

```diff
@@ -33,15 +34,15 @@
 
 #define SAMPLE_RATE 125
 #define BAUD_RATE 115200
-#define INPUT_PIN 36
+#define INPUT_PIN A2
```

Comment:
Always mention that the user has to change this parameter, I think changing `36` to `A0` here means that the code was tested on ESP32 before and now you have tested it with Arduino UNO R3.

Reply:
I have add few comments to describe that we may need to change the input pin number in the Arduino code